instructor/ui/assignment_setup.css

```diff
@@ -904,7 +914,8 @@ input#id_objective{
 
 img.scb_ab_s_small_image_selected {
     float: left;
-    width: 100px;
+    min-width: 100px;
+    min-hight: 100px;
```

Comment:
Same here should be `max-height`

Reply:
@idegtiarov @laleman Checked the UI changes locally, looks better, thanks. Good to merge when above  fixes to CSS are done. :+1: